app/views/plantings/_thumbnail.html.haml

```diff
@@ -38,4 +38,4 @@
       %dd= "#{display_days_before_maturity(planting)}"
 
   .col-xs-9.col-md-8
-    = render partial: 'plantings/planting_progress', locals: {:planting => planting}
\ No newline at end of file
+    = render partial: 'plantings/planting_progress', locals: {planting: planting}
```

Comment:
And another newline here. Hooray for fixing to new syntax while going through! :)